### PR TITLE
enabled ActiveRecord based session store

### PR DESCRIPTION
**Pull request by linki**
... and make it configurable

### CODE REVIEW

File: config/initializers/session_store.rb [hunk 1]
```diff
@@ -29,9 +29,10 @@
 
 # Be sure to restart your server when you modify this file.
 
-session_store = Rails.env.production? ? :cache_store : :cookie_store
+config = OpenProject::Configuration
 
-relative_url_root = OpenProject::Configuration['rails_relative_url_root'].presence
+session_store     = config['session_store'].presence || :cookie_store
+relative_url_root = config['rails_relative_url_root'].presence
```
Comment (myabc) [inline config/initializers/session_store.rb:35]: @linki is there a need for the `.presence` here?

Comment (linki) [inline config/initializers/session_store.rb:35]: shouldn't actually, but while developing i had a case where it returned an empty string which messed up everything. code looks like it's not supposed to be like that but with `presence` i wanted to be on the safe side. how would you do it?

**Comment by linki** [comment]
closed in favour of https://github.com/opf/openproject/pull/1239 thx for your comment, @myabc

**Comment by myabc** [comment]
@linki I'm unable to spot the difference :)

**Comment by linki** [comment]
no change, but different merge target branch

**Pull request closed without merging**